students/shawn_degraw/session03/strformat_lab.py

```diff
@@ -0,0 +1,56 @@
+from decimal import *
+
+#Task One
+print("Task One")
+print("File_{:03d} :  {:.2f}, {:.2e}, {:.3g}".format(2, 123.4567, 10000, 12345.67))
+
+#Task Two
+print("\nTask Two")
+value1=2
+value2=123.4567
+value3=10000
+value4=12345.67
+#?? trying to get significance of 3
+getcontext().prec = 3 
+#??not sure about 3 significant figures for value4
+print(f"File_{str(value1).zfill(3)} :  {round(value2,2)}, {Decimal(value3):.2e}, {Decimal(value4):.2e}")
```

Comment:
Minor: Looks like you've done a lot of research on this! I never knew `.zfill()` method on a string. I think there must be some related f-string syntax that supports 0-padding, but I'm too lazy to look it up... Other things as well, like `round(...)`. Just FYI...